cogs/activity.py

```diff
@@ -288,21 +288,33 @@ async def _generate_chart(
 
     @cog_ext.cog_slash(
         name="trend",
-        description="Dessiner la tendance d'une expression (bêta).",
+        description="Dessiner la tendance d'une expression.",
         guild_ids=guild_ids,
         options=[
             create_option(
                 name="terme",
-                description="Vous pouvez saisir un mot ou une phrase",
+                description="Saisissez un mot ou une phrase.",
                 option_type=3,
                 required=True,
-            )
+            ),
+            create_option(
+                name="periode",
+                description="Période de temps max sur laquelle dessiner la tendance.",
+                option_type=3,
+                required=True,
+                choices=[
+                    create_choice(name="6 mois", value="182"),
```

Comment:
Use int instead of string when discord-py-slash-command will be updated (see [PR](https://github.com/eunwoo1104/discord-py-slash-command/pull/170)).

Reply:
Solved in c9f6e1c1d51395f5915ab07efa5b85bc3fb3ced0